src/pages/RepoPage/FailedTestsTab/FailedTestsPage/FailedTestsTable/FailedTestsTable.tsx

```diff
@@ -222,40 +222,45 @@ const FailedTestsTable = () => {
     testData?.private
 
   const tableData = useMemo(() => {
+    if (!testData?.testResults) return []
+
     return (
-      testData?.testResults.map((result) => {
+      testData.testResults.map((result) => {
         const value = (result.flakeRate ?? 0) * 100
         const isFlakeInt = Number.isInteger(value)
 
+        const flakeRateContent = (
```

Comment:
A this returns JSX. Can you capitalize the first letter so it's FlakeRateContent?